src/Workleap.DomainEventPropagation.Tests/Publishing/EventPropagationPublisherOptionsTests.cs

```diff
@@ -1,29 +1,21 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
-using Moq;
 using Workleap.DomainEventPropagation.Extensions;
 
 namespace Workleap.DomainEventPropagation.Tests.Publishing;
 
 public class EventPropagationPublisherOptionsTests
 {
-    private static readonly Mock<ITopicProvider> TopicProviderMock = new();
-
     [Theory]
     [InlineData(null, null, null)]
-    [InlineData(" ", "whatever", "http://workleap.com")]
-    [InlineData("unrecognizedTopic", "whatever", "http://workleap.com")]
-    [InlineData("Signup", null, "http://workleap.com")]
-    [InlineData("Signup", "whatever", null)]
-    [InlineData("InvalidTopic", "AccessKey", "http://workleap.com")]
-    [InlineData(" ", "AccessKey", "http://workleap.com")]
-    [InlineData(null, "AccessKey", "http://workleap.com")]
-    [InlineData("Organization", "AccessKey", null)]
-    [InlineData("Organization", "AccessKey", "")]
-    [InlineData("Organization", "AccessKey", "AAAAAAABBBBBCCCCCCDEEEEE")]
-    [InlineData("Organization", "  ", "AAAAAAABBBBBCCCCCCDEEEEE")]
-    [InlineData("Organization", null, "http://workleap.com")]
+    [InlineData(" ", "accessKey", "http://topicurl.com")]
```

Comment:
The number of data here was reduced since we didn't have to validate with so many different combinations.